paymentsheet/src/main/java/com/stripe/android/link/repositories/LinkApiRepository.kt

```diff
@@ -186,6 +186,7 @@ internal class LinkApiRepository @Inject constructor(
                 paymentDetailsId = paymentDetails.id,
                 consumerSessionClientSecret = consumerSessionClientSecret,
                 extraParams = extraParams,
+                allowRedisplay = paymentMethodCreateParams.allowRedisplay
```

Comment:
The main change: Passing along the original `allowRedisplay` value to the new Link payment method create params.